openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -11,7 +11,7 @@
   // rules to overcome any poorly defined, scoped, or architected edX LMS-based CSS. Ideally this file will be removed when the LMS Front End is re-visited.
 
   // headers
-  .self-assessment__display__header, .peer-assessment__display__header, .step__header {
+  .self-assessment__display__header, .peer-assessment__display__header, .step__header, .staff-assessment__display__header {
```

Comment:
I think it belongs with these other header overrides because it was just an oversight when staff-assessment__display__header was added that it also wasn't added to this rule.